docs/reference/server-configuration.md

```diff
@@ -439,6 +451,16 @@ The processing semantics to use for persistent queries. The default is
 
 For more information, see [Processing Guarantees](/operate-and-deploy/exactly-once-semantics).
 
+## ksql.streams.rocksdb.config.setter
+
+Assigns a plugin that helps to configure memory allocated to RocksDB across all
```

Comment:
This could technically be configured to use a different kind of `RocksDBConfigSetter` which isn't bounded across all queries, but maybe that's too unlikely to matter for these docs.